### ISSUE TITLE
[impsort] executing job failed: No such file or directory

### ISSUE THREAD

**Issue by AlbertoTejada** [tweekmonster/impsort.vim#32]
After installing the Plugin with Vundle.vim and without adding any configuration in my `.vimrc` file, I got this error when opening a python file (python 3.8):
`[impsort] executing job failed: No such file or directory`
Here is my `.vimrc` file:
 ```
  1 set nocompatible              " required
  2 filetype off                  " required
  3
  4 " Line Numbering
  5 set nu
  6
  7 " Color scheme
  8 syntax on
  9 colorscheme onedark
 10
 11 " Polyglot plugin python highlight configuration:
 12 let g:polyglot_disabled = ['markdown']
 13 let g:python_highlight_all = 1
 14
 15 " Flag white spaces
 16 "au BufRead,BufNewFile *.py,*.pyw,*.c,*.h match BadWhitespace /\s\+$/
 17
 18
 19 " Python PEP8 indentation and folding
 20 au BufNewFile,BufRead *.py:
 21     \ set tabstop=4
 22     \ set softtabstop=4
 23     \ set shiftwidth=4
 24     \ set textwidth=79
 25     \ set expandtab
 26     \ set autoindent
 27     \ set fileformat=unix
 28     "Folding
 29     \ set foldmethod=indent
 30     \ set foldlevel=99
 31
 32 " Enable folding with spacer
 33 nnoremap <space> za
 34
 35 " UTF-8 Support
 36 set encoding=utf-8
 37
 38 " set the runtime path to include Vundle and initialize
 39 set rtp+=~/.vim/bundle/Vundle.vim
 40 call vundle#begin()
 41
 42 " alternatively, pass a path where Vundle should install plugins
 43 "call vundle#begin('~/some/path/here')
 44
 45 " let Vundle manage Vundle, required
 46 Plugin 'gmarik/Vundle.vim'
 47
 48
 49 " add all your plugins here (note older versions of Vundle
 50 " used Bundle instead of Plugin)
 51
 52 " Commentary plugin:
 53 Plugin 'tpope/vim-commentary'
 54 " Git plugin:
 55 Plugin 'tpope/vim-fugitive'
 56 " Static syntax and style checker for Python:
 57 Plugin 'nvie/vim-flake8'
 58 " Another syntax checker (it is not working):
 59 Plugin 'vim-syntastic/syntastic'
 60 " Folding helper for Python:
 61 Plugin 'tmhedberg/SimpylFold'
 62 " Filesystem explorer:
 63 Plugin 'preservim/nerdtree'
 64 " Color themes:
 65 Plugin 'junegunn/seoul256.vim' "Not used at this moment
 66 Plugin 'joshdick/onedark.vim'  "PyCharm like color schema
 67 " Highligthing plugins:
 68 Plugin 'sheerun/vim-polyglot'     "General highlight plugin
 69 Plugin 'tweekmonster/impsort.vim' "Python import highlight
 70
 71 " All of your Plugins must be added before the following line
 72
 73 " NERDTree ignore files:
 74 let NERDTreeIgnore=['\.pyc$', '\~$'] "ignore files in NERDTree
 75
 76 call vundle#end()            " required
 77 filetype plugin indent on    " required
```

**Comment by Nealium** [comment]
**tldr:** `git submodule update --init` somewhere inside `bin/jedi/jedi/third_party` *(exact spot probably doesn't matter, that's just what worked for me)*

This has happened to me multiple times when setting up Vim on different Computers and Operating Systems. I had the *exact* same error as you, but it would occasionally show the directory it looking in: `impsort.vim/bin/jedi/jedi/third_party/typeshed`
Going into that directory showed all third party dirs were empty and running `git submodule update --init`, as mentioned in the readme, filled them & fixed the issue. 

I'm using Vim-Plug, maybe you are too.. I'm gathering that it isn't a *modern plugin manager* lol ..What is odd is if I `rm -rf impsort.vim/` and rerun `:PlugInstall` it installed the submodules correctly, so it'll possibly just be a one time thing

**Comment by Nealium** [comment]
Removing my previous comment as this issue has come back once I started using sessions and not my vim-virtualenv commands.
My problem was `python` wasn't a command as I installed it with `sudo apt install python3`, but it is a command in my virtualenv.

This can be diagnosed by doing `:messages` and `[impsort] /bin/bash: line 1: python: command not found` would be towards the top
* Note: if `'<,'>ImpSort` works, this isn't the issue.

@OP if you're still having this problem, and it isn't what I just described, it might be helpful to add Vim/Neovim version.. it might be some weird async behavior on a particular version
